### ISSUE TITLE
mailroom-ml: CUAD contract corpus is pinned but not wired into enrichment (contract subclass augmentation disconnected)

### ISSUE THREAD

**Issue by Exios66** [LLM-Mailroom-Services/mailroom-issues#113]
**Spawned from #112 (M9a) — corpus EDA (M9a-U3).** Domain: `Datasets` / `Mailroom ML`.

## Problem
`CUAD_FULL_REPO` is revision-pinned in `src/mailroom_ml/config.py:180`:

```
CUAD_FULL_REPO = "Lucius-Morningstar/mailroom-cuad-contracts-full"
CUAD_FULL_REVISION = "e69afe340b48133b7173d74b8ad220fcd28a1a6e"
```

…but it is **not imported anywhere in `training/assemble_enrichment.py`**. The only wired pools are Enron (correspondence), CMS/GNOTHEIA/BDR/INSURBIAS (insurance). So the **contract subclass heads have zero enrichment** — the data-side lever for the weakest head is disconnected.

## Evidence
- `CUAD_FULL_REPO` @ `e69afe3` holds **510 rows (509 with `doc_text`)**, mapped by `metadata.category` through `SUBTYPE_ALIASES` (`src/mailroom_ml/labels.py:66-95`), covering **all 25 canonical contract keys**.
- Rare-tail coverage the head needs: `non_compete_no_solicit` 3, `affiliate` 10, `consulting` 11, `promotion` 12, `reseller` 12, `agency` 13, `transportation` 13, `franchise` 15, `manufacturing` 17, `ip` 17, `marketing` 17, `outsourcing` 18, `supply` 18, `hosting` 20, `co_branding` 22, `joint_venture` 23, `endorsement` 24.
- Contract head run-3 macro-F1 0.0395 (≈ chance over 26 classes) with train support 915 windows / 25 classes; 6 classes <20 windows.

## Impact
The contract head cannot be improved data-side until CUAD is wired. M9a-U3 concluded the loss-side test comes first, but CUAD is the ready-made contract lever and is currently dead config.

## Proposed fix
1. Wire a `_run_pool`/adapter for `CUAD_FULL_REPO` in `assemble_enrichment.py`, mapping `metadata.category` → canonical key via `SUBTYPE_ALIASES`.
2. Define the governing cap policy (CUAD is currently **uncapped** — no tier entry).
3. Add the dedup-vs-canonical step (`_dedup_vs_canonical` against `mailroom-finetune@19720ceb`).

## Blocked by
The allocator bug (#spawned separately) — with the current greedy `mixture_caps` allocator, adopting CUAD would deliver 0 allowance to alphabetically-late contract subclasses regardless.

Relates to #112, #85.

**Comment by Exios66** [comment]
Repo-side wiring landed on `mailroom-ml` `main` (`assemble_cuad_pool`, `POOL_FLAGS` cuad, tests in `tests/test_enrichment.py`). Close-out + governance sync tracked at https://github.com/LLM-Mailroom-Services/mailroom-ml/issues/14 — please verify and close here with test/SHA evidence.

**Comment by Exios66** [comment]
## Proof-close — CUAD Tier-1 wiring on `mailroom-ml` main

Repo comment + live code confirm the structural hazard is fixed; no new PR.

**Landing commit:** [`ecc850e8f0f36e7f98cd9a838c96b709a506e333`](https://github.com/LLM-Mailroom-Services/mailroom-ml/commit/ecc850e8f0f36e7f98cd9a838c96b709a506e333) — `assemble_cuad_pool` + related enhancements  
**Live `main` tip (contains the above):** [`e3dc3294635c03e753712f8ad8f57a2b2ca98728`](https://github.com/LLM-Mailroom-Services/mailroom-ml/commit/e3dc3294635c03e753712f8ad8f57a2b2ca98728)

### Evidence
- `src/mailroom_ml/enrichment.py` — `assemble_cuad_pool` present
- `training/assemble_enrichment.py` — Tier-1 wires `"cuad": (assemble_cuad_pool, …)` / `--cuad-pool`
- Tests: `test_cuad_pool_maps_category_and_head_checks`, `test_cuad_pool_caps_and_dedups_vs_canonical` (+ CLI Tier-1 CUAD paths)

Governance board refresh tracked at mailroom-ml #14 (docs only).

Closing.

— Mailroom Eng
